syntax = "proto3";

package eu.surething_project.core.wi_fi;

option java_multiple_files = true;
option java_package = "eu.surething_project.core.wi_fi";
option java_outer_classname = "WiFiProto";

// Wi-Fi networks evidence 
message WiFiNetworksEvidence {
    // unique id
    string id = 1;

    // Access point (AP) definition
    message AP{
        // AP identifier
        string ssid = 1;
        // measured RSSI 
        string rssi = 2;
    }

    // detected APs
    repeated AP aps = 2;
}
